plugins/procfs/README.md

```diff
@@ -1,3 +1,153 @@
+|`kernel_new_forks`|CounterDiff|none|Number of forked operations : When a process creates a copy of itself|LocalMachine|LocalMachine||
+|`kernel_n_procs_running`|Gauge|none|Number of processes in a runnable state|LocalMachine|LocalMachine||
+|`kernel_n_procs_blocked`|Gauge|none|Numbers of processes that are blocked on input/output operations|LocalMachine|LocalMachine||
+|`cpu_time_delta`|CounterDiff|nanosecond|CPU usage|LocalMachine|Process|[kind](#kind)|
+|`memory_usage`|Gauge|bytes|Memory usage|LocalMachine|Process|[kind](#kind)|
+
+And in case we configure the plugin with the collect of detailed memory metrics :
+
+|Name|Type|Unit|Description|Ressource|RessourceConsumer|Attributes|
+|----|----|----|-----------|---------|-----------------|----------|
+|`mem_total`|Gauge|bytes|Total amount of physical memory available in the system|LocalMachine|LocalMachine||
+|`mem_free`|Gauge|bytes|Refers to truly free memory, which is not used by processes, caches or buffers|LocalMachine|LocalMachine||
+|`mem_available`|Gauge|bytes|More accurate estimate of the actual available memory, taking into account free memory and also caches and buffers that can be freed|LocalMachine|LocalMachine||
+|`cached`|Gauge|bytes|Cached is the memory used for ROM cache (page cache) in memory|LocalMachine|LocalMachine||
+|`swap_cached`|Gauge|bytes|Memory that was placed in swap and brought back to RAM, but remains in swap file|LocalMachine|LocalMachine||
+|`active`|Gauge|bytes|Memory that has been used recently|LocalMachine|LocalMachine||
+|`inactive`|Gauge|bytes|Used memory that can be reclaimed, freed, or moved to swap.|LocalMachine|LocalMachine||
+|`mapped`|Gauge|bytes|Memory used for files associated with processes in memory, file and memory mapping.|LocalMachine|LocalMachine||
+
+### Attributes
+
+#### Kind
+
+The kind of the memory is the allocated memory space reserved by the system or the hardware (<https://man7.org/linux/man-pages/man5/proc_pid_status.5.html>) :
+
+|Value|Description|
+|-----|-----------|
+|`resident`|Resident set size, measured in pages (same as VmRSS in `/proc/<pid>/status`)|
+|`shared`|number of resident shared pages (i.e., backed by a file) (same as RssFile+RssShmem in `/proc/<pid>/status`)|
+|`virtual`|Virtual memory size (same as VmSize in `/proc/<pid>/status`)|
+
+The kind of the CPU time delta is the average CPU time spent by various tasks :
+
+|Value|Description|
+|-----|-----------|
+|`user`|CPU time spent by the process in user mode|
+|`system`|CPU time used by the system (Kernel)|
+|`guest`|CPU time used by a guest virtual machine when a hypervisor virtualizes environments|
+
+#### cpu_state
+
+The CPU states correspond the amount of time, measured in ticks, the CPU has been in specific states. These fields are measured in ticks because the underlying data from the kernel is measured in ticks (<https://docs.rs/procfs-core/latest/procfs_core/struct.CpuTime.html>) :
```

Comment:
1. The metric in Alumet is in `millisecond`: [see code](https://github.com/alumet-dev/alumet/blob/c3efb9d9f82dbe3c4c88900966e605076e442a5e/plugins/procfs/src/kernel.rs#L55-L56)
2. `cpu_state` is not an amount of time, it's an attribute that indicates the kind of cpu time that is measured